src/components/v2/chartDeploymentHistory/ChartDeploymentHistory.component.tsx

```diff
@@ -459,6 +459,12 @@ function ChartDeploymentHistory({
                                     Moment12HourFormat,
                                 )}
                             </time>
+                            {deployment.deployedBy && (
```

Comment:
Is it possible to have deployedBy empty, if it is then put deployed?.deployedBy

Reply:
done